internal/collector/event.go

```diff
@@ -68,7 +68,7 @@ func (collector *EventCollector) Run(stopCh <-chan struct{}) {
 		AddFunc: func(obj interface{}) {
 			ev := obj.(*v1.Event)
 			// Count only Events created after the exporter starts running.
-			if beforeLastEvent(startRunning, ev) {
+			if beforeLatestEvent(startRunning, ev) {
 				// FIXME: take into account the event count.
```

Comment:
Is this still a valid TODO?

Reply:
No, I removed it. Thanks for noticing it :)